test-data/unit/cmdline-based.test

```diff
@@ -0,0 +1,30 @@
+[case testIgnoreMissingPyTyped]
+# cmd: mypy main.py
+[file main.py]
+from a import x
+from b import y
+[file mypy.ini]
+\[mypy]
+\[mypy-a]
+ignore_missing_py_typed = true
+[file pypath/a/__init__.py]
+x = 0
+[file pypath/b/__init__.py]
+y = 0
+[out]
+main.py:2:1: error: Skipping analyzing "b": module is installed, but missing library stubs or py.typed marker  [import]
```

Comment:
How do the other tests work? Wouldn't you have to specify a `py.typed` for every single other test?

Reply:
mypy_path vs python_path